<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeedtoTime4</class>
 <widget class="QMainWindow" name="SeedtoTime4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 4 Seed to Time</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QTabWidget &gt;  QWidget &gt; QWidget {
	background-color: #E6E6E6;
}

QTabBar::tab {
	color: black;
	background-color: #A1ABB3;
	font-family: &quot;Myriad Pro&quot;;
}

QTabBar::tab:selected {
	color: white;
	background-color: #1D303C;
}

QCheckBox::indicator {
	border: 2px solid white;
	border-radius: 3px;
	height: 10px;
	width: 10px;
}

QCheckBox::indicator:unchecked {
	background-color: #274050;
}

QCheckBox::indicator:checked {
	background-color: white;
}

QTableView {
	border: 4px solid DodgerBlue;
}

QHeaderView::section {
	border: none;
	background-color: #274050;
	padding-top: 3px;
	padding-bottom: 3px;
	color: white;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
}

QGroupBox {
	background-color: #274050;
	border-radius: 10px;
	margin-top: 7px;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

.QGroupBox * {
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QLabel {
	color: white;
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 20px;
	padding: 3px 5px 3px 5px;
	background-color: #969F91;
	border-radius: 5px;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDPPt">
       <attribute name="title">
        <string>DPPt</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Calibration</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="5">
            <widget class="QLineEdit" name="lineEditPlusSecondsDPPt">
             <property name="text">
              <string notr="true">1</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Second</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string notr="true">+</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditMinusDelayDPPt">
             <property name="text">
              <string notr="true">10</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="pushButtonSearchFlips">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Search Flips</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QPushButton" name="pushButtonCalibrateDPPt">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string notr="true">-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string notr="true">-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEditMinusSecondsDPPt">
             <property name="text">
              <string notr="true">1</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEditPlusDelayDPPt">
             <property name="text">
              <string notr="true">10</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string notr="true">+</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="9">
            <widget class="TableView" name="tableViewDPPtCalibrate"/>
           </item>
           <item row="1" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Seed to Time</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxSeedDPPt">
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="8">
            <widget class="TableView" name="tableViewDPPt"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Seed</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEditYearDPPt">
             <property name="text">
              <string notr="true">2000</string>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEditSecondsDPPt">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Second</string>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QPushButton" name="pushButtonGenerateDPPt">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLabel" name="labelCoinFlips">
             <property name="text">
              <string>Coin Flips:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="checkBoxSecondsDPPt">
             <property name="text">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHGSS">
       <attribute name="title">
        <string>HGSS</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Seed to Time</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0" colspan="3">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Year</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Second</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Seed</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QTextBox" name="textBoxSeedHGSS">
               <property name="maxLength">
                <number>8</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEditYearHGSS">
               <property name="text">
                <string notr="true">2000</string>
               </property>
               <property name="maxLength">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QCheckBox" name="checkBoxSecondsHGSS">
               <property name="text">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLineEdit" name="lineEditSecondsHGSS">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maxLength">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QPushButton" name="pushButtonMap">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>18</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color: orange;
}</string>
               </property>
               <property name="text">
                <string>Map</string>
               </property>
              </widget>
             </item>
             <item row="1" column="6">
              <widget class="QCheckBox" name="checkBoxR">
               <property name="text">
                <string>R</string>
               </property>
              </widget>
             </item>
             <item row="1" column="7">
              <widget class="QLineEdit" name="lineEditR">
               <property name="maxLength">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="8">
              <widget class="QCheckBox" name="checkBoxE">
               <property name="text">
                <string>E</string>
               </property>
              </widget>
             </item>
             <item row="1" column="9">
              <widget class="QLineEdit" name="lineEditE">
               <property name="maxLength">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="10">
              <widget class="QCheckBox" name="checkBoxL">
               <property name="text">
                <string>L</string>
               </property>
              </widget>
             </item>
             <item row="1" column="11">
              <widget class="QLineEdit" name="lineEditL">
               <property name="maxLength">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelElmCalls">
             <property name="text">
              <string>Elm Calls:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButtonGenerateHGSS">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelRoamers">
             <property name="text">
              <string>Roamers:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="TableView" name="tableViewHGSS"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Calibration</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string notr="true">+</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Second</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="lineEditPlusSecondsHGSS">
             <property name="text">
              <string notr="true">1</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QPushButton" name="pushButtonSearchCalls">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Search Calls</string>
             </property>
            </widget>
           </item>
           <item row="1" column="8">
            <widget class="QPushButton" name="pushButtonCalibrateHGSS">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string notr="true">-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditMinusDelayHGSS">
             <property name="text">
              <string notr="true">10</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEditMinusSecondsHGSS">
             <property name="text">
              <string notr="true">1</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEditPlusDelayHGSS">
             <property name="text">
              <string notr="true">10</string>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string notr="true">-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string notr="true">+</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="9">
            <widget class="TableView" name="tableViewHGSSCalibrate"/>
           </item>
           <item row="1" column="6">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>textBoxSeedDPPt</tabstop>
  <tabstop>lineEditYearDPPt</tabstop>
  <tabstop>lineEditSecondsDPPt</tabstop>
  <tabstop>pushButtonGenerateDPPt</tabstop>
  <tabstop>tableViewDPPt</tabstop>
  <tabstop>lineEditMinusDelayDPPt</tabstop>
  <tabstop>lineEditPlusDelayDPPt</tabstop>
  <tabstop>lineEditMinusSecondsDPPt</tabstop>
  <tabstop>lineEditPlusSecondsDPPt</tabstop>
  <tabstop>pushButtonSearchFlips</tabstop>
  <tabstop>pushButtonCalibrateDPPt</tabstop>
  <tabstop>tableViewDPPtCalibrate</tabstop>
  <tabstop>textBoxSeedHGSS</tabstop>
  <tabstop>lineEditYearHGSS</tabstop>
  <tabstop>checkBoxSecondsHGSS</tabstop>
  <tabstop>lineEditSecondsHGSS</tabstop>
  <tabstop>pushButtonGenerateHGSS</tabstop>
  <tabstop>tableViewHGSS</tabstop>
  <tabstop>lineEditMinusDelayHGSS</tabstop>
  <tabstop>lineEditPlusDelayHGSS</tabstop>
  <tabstop>lineEditMinusSecondsHGSS</tabstop>
  <tabstop>lineEditPlusSecondsHGSS</tabstop>
  <tabstop>pushButtonSearchCalls</tabstop>
  <tabstop>pushButtonCalibrateHGSS</tabstop>
  <tabstop>tableViewHGSSCalibrate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
